packages/terra-responsive-element/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-responsive-element",
+  "main": "lib/ResponsiveElement.js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "The terra-responsive-element conditionally renders components based on viewport size",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-responsive-element",
+    "ResponsiveElement",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "resize-observer-polyfill": "^1.4.1",
```

Comment:
`resize-observer-polyfill` is imported in `src`. We need to move this to a dependency so it is installed when an application consumes responsiveElement.